lib/src/base-styles/_nx-colors.scss

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+:root {
+  --nx-color-border-default: #{$nx-indigo-100};
+  --nx-color-border-subsection: #{$nx-grey-200};
+  --nx-color-border-form-element: #{$nx-grey-1100};
+  --nx-color-border-form-element-pristine: #{$nx-indigo-100};
+  --nx-color-border-disabled: #{$nx-grey-500};
+  --nx-color-border-focus: #{$nx-teal-300};
+
+  --nx-color-box-shadow-focus: #{transparentize($nx-teal-300, 0.7)};
+
+  --nx-color-background-component: #{$nx-white};
+  --nx-color-background-site: #{$nx-grey-50};
+  --nx-color-background-selected: #{$nx-blue-200};
+  --nx-color-background-active: #{$nx-blue-200};
+  --nx-color-background-hover: #{$nx-grey-100};
+  --nx-color-background-disabled: #{$nx-grey-200};
```

Comment:
These are too vague IMO, they could be the background of anything. Maybe something like `nx-color-interactive-selected` etc?

Reply:
Per my comment here: https://github.com/sonatype/sonatype-react-shared-components/pull/383/files#r633547053 and also on the focus states, maybe:

`--nx-color-element-background-disabled` etc.

Reply:
> Per my comment here: https://github.com/sonatype/sonatype-react-shared-components/pull/383/files#r633547053 and also on the focus states, maybe:
>
> --nx-color-element-background-disabled etc.

Addressed in other thread: https://github.com/sonatype/sonatype-react-shared-components/pull/383#discussion_r635581410 7a3d082